FlightsWebsite/src/js/weather.js

```diff
@@ -0,0 +1,64 @@
+function selectedWeather() {
+    const option = document.getElementById("weather");
+    switch (option.selectedIndex) {
+        case 5:
+            displayGraphNoOfObservationsPerOrigin();
+            break;
+        default:
+            console.log("Selected dropdown in weather:" + option.selectedIndex)
+    }
+
+}
+
+function displayGraphNoOfObservationsPerOrigin() {
+    fetch('http://localhost:8080/weather/getNumberOfWeatherObservationsByOrigin')
```

Comment:
Don't hardcode localhost in the URL. This will fail on the cloud. Also, it's good idea to make URL as some const variable, so that you need to change it in only one place

Reply:
yeah I see .. so where we will put the configuration .. which path to choose (either localhost or cloud) ? .. maybe we can use somehow the variables on gcloud

Reply:
Yyy, we could, but I don't know how exactly to do that. And it will require some configuration form our side also. I don't think it's worth to do that for just one variable. Just remember to set it to correct URL before making PR

Reply:
okay (Y)